ipalib/__init__.py

```diff
@@ -977,7 +977,7 @@ def create_api(mode='dummy'):
     api = API()
     if mode is not None:
         api.env.mode = mode
-    assert mode != 'production'
+    assert mode != f'production'
```

Comment:
This doesn't need to be a format string IMHO.

Reply:
I made it an f-string to trigger syntax error on Python 2. It was easier than adding a version check.